src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -80,20 +80,22 @@ public void createFeedbackSessions(Collection<FeedbackSessionAttributes> feedbac
         Iterator<FeedbackSession> it = endTimeEntities.iterator();
 
         while (it.hasNext()) {
-            startCal.setTime(start);
-            endCal.setTime(end);
-            FeedbackSessionAttributes fs = new FeedbackSessionAttributes(it.next());
-            
-            Date standardStart = TimeHelper.convertToUserTimeZone(startCal, fs.timeZone - zone).getTime();
-            Date standardEnd = TimeHelper.convertToUserTimeZone(endCal, fs.timeZone - zone).getTime();
-            
-            if((fs.startTime != null && fs.startTime.getTime() >= standardStart.getTime() && fs.startTime.getTime() < standardEnd.getTime())                    
-                ||(fs.endTime != null && fs.endTime.getTime() > standardStart.getTime() && fs.endTime.getTime() <= standardEnd.getTime())) {
-            
-                list.add(fs);
-            
+            FeedbackSession feedbackSession = it.next();
+            if (!JDOHelper.isDeleted(feedbackSession)) {
+                startCal.setTime(start);
+                endCal.setTime(end);
+                FeedbackSessionAttributes fs = new FeedbackSessionAttributes(it.next());
```

Comment:
This should be <code>feedbackSession</code> here right, rather than it.next() again